SlevomatCodingStandard/Sniffs/TypeHints/TypeHintDeclarationSniff.php

```diff
@@ -29,6 +31,10 @@ class TypeHintDeclarationSniff implements \PHP_CodeSniffer_Sniff
 
 	const CODE_MISSING_TRAVERSABLE_RETURN_TYPE_HINT_SPECIFICATION = 'MissingTraversableReturnTypeHintSpecification';
 
+	const CODE_USELESS_PARAMETER_DOC_COMMENT = 'UselessParameterDocComment';
```

Comment:
I think `CODE_USELEESS_PARAMETER_ANNOTATION` would be better. The same for the second constant.